Art/oOXxTNTxXOo-colorchange/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Word Gravity</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
# Incorrect CSS file name > reflected in this href
- [ ] Please change the name of the css file as described inn the readme and the checklist
- [ ] Then you could update your html link ref here accordingly